### PR TITLE
Teeworlds.game version 0.7.5

### PR DESCRIPTION
**Pull request by Twig6943**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100693)

### CODE REVIEW

File: manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Teeworlds.game
+PackageVersion: 0.7.5
+Installers:
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml:9]:
```suggestion
  NestedInstallerType: portable
```

Comment (stephengillie) [reply manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml:9]: Currently, only the server is installed, and the game client is mistaken for the installer and becomes removed.

**Comment by stephengillie** [comment]
Hi @Twig6943,

The application installed normally, but the client gave an error instead of launching:
> [2023-03-27 13:50:27][console]: failed to open 'autoexec.cfg'

Is there a command to generate this file?

File: manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml (hunk 1 above)
Comment (Twig6943) [reply manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml:9]: sorry but I don't know how to fix that

Comment (stephengillie) [reply manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml:9]: Oh, sorry. Approving the above suggestion will fix that. I've tested this change in a VM, and it resulted in the application being installed.

**Pull request closed without merging**